src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -421,15 +421,15 @@ func (d *Dumper) getPodSummary(resource, podName, crName string, namespace strin
 		}
 		ports = port + ":3306"
 		summCmdName = "pt-mysql-summary"
-		summCmdArgs = []string{"--host=127.0.0.1", "--port=" + port, "--user=root", "--password='" + string(pass) + "'"}
-	case "pg","pgv2":
-		var kubeconfig string=""
-		if d.kubeconfig != "" { 
-			kubeconfig=" --kubeconfig=" + d.kubeconfig 
+		summCmdArgs = []string{"--host=127.0.0.1", "--port=" + port, "--user=root", "--password=" + string(pass)}
+	case "pg", "pgv2":
+		var kubeconfig string = ""
```

Comment:
Explicit assigning to empty string is redundant. String variables initialized to empty string by deafult.
```suggestion
		var kubeconfig string
```